code/datums/components/deployable_item.dm

```diff
@@ -8,13 +8,16 @@
 	var/obj/deploy_type
 	///Any extra checks required when trying to deploy this item
 	var/datum/callback/deploy_check_callback
+	///Helps to determine if the item should be deployable in areas like the tad and alamo
+	var/restricted_deployment = FALSE
 
-/datum/component/deployable_item/Initialize(_deploy_type, _deploy_time, _undeploy_time, _deploy_check_callback)
+/datum/component/deployable_item/Initialize(_deploy_type, _deploy_time, _undeploy_time, _restricted_deployment, _deploy_check_callback)
```

Comment:
you make me extremely nervous when you add an arg thats not at the end, yet you haven't changed a single instance of the component in code.
I am almost 100% certain you are breaking other objects.

Reply:
Alright, moved it back. I just forgot null existed to be able to go past the _undeploy_time arg